opendatahub-io/data-science-pipelines-operator
# WIP: Dynamically pull in manifest templates

## The issue resolved by this Pull Request:
Resolves #201

## Description of your changes:
Rather than explicitly listing out every template file a component needs to apply/manage, this feature will dynamically read a target directory and applies *all* templates in that folder (currently, only in that level i.e. no recursion)

## Testing instructions
Run DSPO, create a DSPA (preferably with all optional deployments such as MLMD, APIServer.EnableRoute, etc turned on), and ensure the DSPA deploys correctly

## Checklist
